store/tikv/2pc.go

```diff
@@ -26,7 +26,28 @@ import (
 	"golang.org/x/net/context"
 )
 
-type txnCommitter struct {
+type twoPhaseCommitAction int
+
+const (
+	actionPrewrite twoPhaseCommitAction = 1
+	actionCommit   twoPhaseCommitAction = 2
+	actionCleanup  twoPhaseCommitAction = 3
+)
+
+func (ca twoPhaseCommitAction) String() string {
+	switch ca {
+	case actionPrewrite:
+		return "prewrite"
+	case actionCommit:
+		return "commit"
+	case actionCleanup:
+		return "cleanup"
+	}
+	return "unknown"
+}
+
+// twoPhaseCommitter executes a two-phase commit protocol.
+type twoPhaseCommitter struct {
```

Comment:
It's 2PC protocol,  '2' is a digit, can't be used at first character.
We could do one-phase commit optimization for single region transaction in the future.